### DISCUSSION TITLE
Link the loaded course to the attachment, not the tractor.

### DISCUSSION THREAD

**Discussion by JustDen1234** [Courseplay/Courseplay_FS22#2611] [Ideas]
We all use one tractor for multiple attachments. And attachments are the base of the course. So it would be nice that the course is 'linked' to the attachment, not to the tractor. 

Maybe explain with an example.
I have one field that I use to create hay.
1. Add mower to the tractor, clear the course, load the course for mowing, start mowing
2. Add tedder to the tractor, clear the course, load the course for teddering, start teddering
3. Add windrower to the tractor, clear the course, load the source for windrowing, start windrowing
4. Add forage wagon to the tractor, clear the couser, load the source for getting the hay of the field, get the hay of the field

If the course would be linked to the attachment, this would be the work:
1. Add mower to the tractor, start mowing
2. Add tedder to the tractor, start teddering
3. Add windrower to the tractor, start windrowing
4. Add forage wagon to the tractor, get the hay of the field

I hope this is possible.

**Comment by Tensuko** [comment]
We have plans to get some settings linked to the tools so you always have to correct settings and don't need to switch.
But we will not do that for the course.
To much data that we would need to store in the savegame and that also leads to Problems in the MP Synchronisation.
Also all our system is based on the root vehicle. (Tractor, Combine, etc).
Also what about vehicles that don't have anything to attach to, would need a seperation just for this.

I don't think that a few delete and load again is that much of a problem compared to this rly big change we would have to TRY and see if it works.

So this won't happen.
